cmd/syncthing/main.go

```diff
@@ -144,6 +144,7 @@ type serveOptions struct {
 	AuditFile        string `name:"auditfile" placeholder:"PATH" help:"Specify audit file (use \"-\" for stdout, \"--\" for stderr)"`
 	BrowserOnly      bool   `help:"Open GUI in browser"`
 	DataDir          string `name:"data" placeholder:"PATH" env:"STDATADIR" help:"Set data directory (database and logs)"`
+	TempDir          string `name:"tmp" placeholder:"TMPDIR" env:"STTMPDIR" help:"Set single tempdir for storing incomplete files (optional)"`
```

Comment:
You've got a mix of "temp" and "tmp" in many different places. Maybe streamline everything to just "temp" (and "TEMPDIR", "STTEMPDIR", "tempdir", etc.)?

Reply:
Sure, not an issue